packages/Popover/src/hooks.ts

```diff
@@ -56,3 +56,18 @@ export function useMutationObserver<Value>(
 
   return value;
 }
+
+export function useElementNode<ElementType = HTMLElement>(): [
+  ElementType | null,
+  (node: ElementType | null) => void
+] {
+  const [element, setElement] = useState<ElementType | null>(null);
+
+  const elementRefCallback = useCallback(node => {
+    if (node !== null) {
```

Comment:
Any reason not to do
``` js
if (node) {
```
instead?

Reply:
Copied from [the react docs FAQ](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node). Don't really care either way.

Reply:
I don't personally see a reason not to do `if (node) {` and it's a bit cleaner.